packages/ui/cypress/tests/watched-accounts.cy.ts

```diff
@@ -239,4 +239,103 @@ describe('Watched Accounts', () => {
         multisigPage.reviewButton().should('not.exist')
       })
   })
+
+  it('can see the same details displayed when watching a pure or its multisig address', () => {
+    const { purePublicKey, address: multisigAddress } = watchMultisigs['multisig-with-pure']
+
+    cy.visitWithLocalStorage({
+      url: landingPageUrl,
+      watchedAccounts: [purePublicKey]
+    })
+    // store details of the displayed addresses for later comparison
+    multisigPage.accountHeader().within(() => {
+      accountDisplay.addressLabel().should('be.visible').invoke('text').as('headerPureAddress')
```

Comment:
What I preferred about using the aliases is that when we were inspecting the UI and watching via the pure we were dynamically grabbing and storing whatever data was displayed (without caring, just knowing this is the view when a pure is watched) and then only asserting when we watch via the multisig that the displayed details are the same as what we saw previously. 

I've updated now to compare using the static values as requested but this means we are now first asserting on the display when watching a pure key (already covered in other tests) and then again the displayed data when watching a multisig key, a trivial difference that probably makes no change in execution time but it's less of a well-written test in my opinion. We break test writing conventions of only asserting after the point of change (but admittedly we do this all the time especially with `should('be.visible')` checks because we have to 😄). 

Ultimately if it reads better I guess that matters more but I wanted to explain my view. I still think cypress alias will have their place when checking dynamic data but perhaps it makes less sense when values are static like this (and I do agree the @ syntax is a little odd!).

Reply:
Yep this reads well, thanks :)